### PR TITLE
Add case to GetDistro() for CentOS linux.

### PR DESCRIPTION
**Pull request by andrewkrug**
Operating system detection for setting up server.yml non functional on CentOS to spite binary compatibility.  Added a case for this to GetDistro().

### CODE REVIEW

File: lib/distro_entry.py [hunk 1]
```diff
@@ -29,6 +29,8 @@ def GetDistro():
       return "debian"
     if distribution in ["red hat enterprise linux server"]:
       return "redhat"
+    if distribution in ["centos linux"]:
```
Comment (grrrrrrrrr) [inline lib/distro_entry.py]: cool thanks! Do you mind putting the centos string in the list right above since we have it already?

Comment (andrewkrug) [inline lib/distro_entry.py]: Not a problem.  Just made that change.

**Comment by andrewkrug** [comment]
Looks like the travis build failed only due to some rekall tests that aren't currently working.  Let me know if that is indeed the case.

**Comment by grrrrrrrrr** [comment]
Yes, the test failures are not related, they are caused by the outdated Rekall version. Thanks!

**Pull request merged**